### ISSUE TITLE
[Q] Image doesn't show in wandb.ai

### ISSUE THREAD

**Issue by Yoonho-Na** [wandb/wandb#5165]
<img width="640" alt="스크린샷 2023-03-15 오후 6 50 13" src="https://user-images.githubusercontent.com/53941554/225272490-d98850cb-7ce5-4ef7-aa32-e99240f5b9b4.png">

I have no idea why it says select runs.

It is already selected

**Comment by nate-wandb** [comment]
Hi @Yoonho-Na, could you send me a link to the workspace where you are seeing this?

Thank you,
Nate

**Comment by Yoonho-Na** [comment]
@nate-wandb Hi

I found what's wrong with the code.

making log directory before wandb init solved the problem.
(I think wandb logged in the tmp directory before)

I suggest to add some lines such as 'logging in tmp dir may not show results.'

It only says select runs in the website and 

`wandb: WARNING Path logs/custom_dir/wandb/ wasn't writable, using system temp directory.
`
in terminal.

So I didn't notice that this would cause not showing the images and took a while to fix it.

anyways since I solved what was wrong, I close this issue.

thanks.
